hermes-guide-days-website/src/app/app.module.ts

```diff
@@ -3,16 +3,13 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { FooterComponent } from './footer/footer.component';
 
 @NgModule({
-  declarations: [
-    AppComponent
-  ],
-  imports: [
-    BrowserModule,
-    AppRoutingModule
-  ],
+  declarations: [AppComponent, FooterComponent],
+  imports: [BrowserModule, AppRoutingModule, BrowserAnimationsModule],
```

Comment:
one under the other, not inline, also any reason for adding BrowserAnimationsModule?